modules/deployment/src/main/java/org/eclipse/xpanse/modules/deployment/DeployService.java

```diff
@@ -126,6 +122,29 @@ public DeployTask createNewDeployTask(DeployRequest deployRequest) {
         return deployTask;
     }
 
+    private void checkContext(ServiceTemplateEntity existingServiceTemplate,
```

Comment:
which context are we validating here? Probably we need a better name here. 
looking at what we are doing here, the same validations are needed or already used in new deployment as well. We must have this at a common place.